docs/content/dashboard.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Dashboard
+layout: docs
+toc_min_level: 2
+---
+
+# Dashboard
+
+Flynn comes with a web dashboard that supports managing apps and databases
+running on Flynn. After installing Flynn, you can reach the dashboard at
+`https://dashboard.$CLUSTER_DOMAIN`.
+
+An ephemeral CA certificate is used to secure access to the dashboard, for more
+details look at [the security
+documentation](/docs/security#dashboard-ca-certificate).
+
+## GitHub Integration
+
+The dashboard supports deploying apps directly from GitHub, without touching the
+command line. To link a GitHub account to the dashboard, follow the instructions
+to generate an API token and add it to the dashboard.
```

Comment:
The last sentence isn't clear on where the instructions are. Maybe change to something like "follow the built in instructions"?